### ISSUE TITLE
Parametric bootstrapping question

### ISSUE THREAD

**Issue by bradduthie** [StirlingCodingClub/studyGroup#26]
How could we write a script for parametric bootstrapping in R?

I've tried the following:

```
test <- bootstrapR(data)
```

And that did not work. Do you know how to do this @rosemckeon

**Comment by jejoenje** [comment]
Was parametric bootstrapping already covered in @bradduthie session [Introduction to randomisation, bootstrap, and Mote Carlo methods in R](https://stirlingcodingclub.github.io/randomisation/randomisation_notes.html) from last year?
The slides etc are in the link above... if not, or it needs expanding on, please let us know...?
We could certainly re-run that session?

**Comment by bradduthie** [comment]
Yep! I think we covered it @jejoenje in the randomisation topic last year. I would definitely be happy to do another session on it!
